doc/source/Tutorials/GettingStarted/index.md

```diff
@@ -1,407 +1,499 @@
-	- Type `/group/camera` into the camera text field.
+2. Right-click. The node creation menu will open.
 
-Next we need to define what images we want to output :
+> Tip :
+> With the cursor hovering over the _Graph Editor_, you can also hit <kbd>Tab</kbd> to open the node creation menu.
 
-- Select the Outputs node.
-- Use the NodeEditor to add an output, by _Left-Clicking_ on the ![Plus icon](images/plus.png) and choosing the _/Interactive/Beauty_ item from the popup menu.
+3. Type `Camera`. A list of search results will appear. By default, _Camera_ will be highlighted.
 
-Now we can start the renderer :
+4. Hit <kbd>Enter</kbd> to create a new Camera node.
 
-- Select the InteractiveAppleseedRender node.
-- Use the NodeEditor to set the renderer state to running.
+    ![A new Camera node](images/mainCameraNode.png "A new Camera node")
 
-And finally we can view the result :
+As before, the newly created node will be selected automatically, and the _Viewer_, _Scene Hierarchy_, and _Node Editor_ will update to reflect this new selection.
 
-- Select the Catalogue node.
-- Move the mouse into the Viewer and press 'f' to frame the image.
 
-![First render](images/firstRender.png)
+## Nodes, Scenes, and Plugs ##
 
-It's hardly worthy of an Oscar, but we've successfully made our first image, and are in a good place to start adding some lighting and shading.
+So far, your script is set up as such: the SceneReader node is outputting a scene containing Gaffy's geometry, and the Camera node is outputting a scene containing a camera. Since neither node is connected, each scene remains separate. This is because each node in Gaffer outputs an entire self-contained scene hierarchy at its position in the graph. If they do not join somewhere later in the graph, the two scenes will never interface with each other. A key part to understand about the node graph is that:
 
-Organising the user interface
------------------------------
+> Important :
+> Each node contains either the scene that has been passed to it through a plug, or the scene that it creates.
 
-Before we dive into lighting and shading though, let's take a brief detour to reorganise the user interface a little better for our workflow. As we've already learned, editors such as the Viewer, NodeEditor and HierarchyView always show the currently selected node by default. This isn't always convenient, because often we want to edit one node while viewing the effects in another. This can be achieved by "pinning" specific nodes into an editor, so that they stay there regardless of the selection :
+In short, the scene is not a single hierarchy that exists in the background of the graph, but is instead carried by the nodes. You can test this by clicking the background of the _Graph Editor_, and observing that once the you have no node selected, the _Scene Hierarchy_ goes blank. This will become very relevant when you begin combining scenes together and selectively modifying a scene's locations.
```

Comment:
It might be a conceit, but I think "carried" makes some sense. I think it's clear that nodes modify (compute) parts of the scene, but the point I'm trying to work toward is:
1. The scene does _not_ simply get imported by the reader, broken down, modified, and then appear as a final result at the end of the graph. Each node holds/contains/carries/keeps a full scene, with full hierarchy and properties.
2. Each node only has access to the instantiated scene it was provided, usually by other nodes.

This was important to my understanding, because the scene nodes behave unlike analogous signal processing or compilation paradigms, where the final result is inaccessible until a final bake. Maybe to make this clearer I should clarify and outright say "each node contains a full scene", but I didn't want to delve too deep into that territory yet.

Reply:
> Each node holds/contains/carries/keeps a full scene, with full hierarchy and properties.

That's not what's happening though. That could imply an unsustainable memory spike (eg each node duplicates and stores the input scene). I think its an important statement to make that Gaffer is explicitly not doing this. Some older VFX software did do that and its part of why they've either died off or moved onto better approaches.

In Gaffer, the node isn't holding onto anything, its re-computing individual locations of the scene each time its asked to do so. The backend computation process (that we should definitely not be describing here) does some holding of things so the nodes can fast-fetch previously computed results. But the nodes themselves don't hold anything other than the values of their unconnected input plugs (well, not even that... the plugs hold the values and the nodes hold the plugs).

If compute is no good, how about "but is instead **_output_** by the **_individual_** nodes"?